src/main/java/com/alipay/sofa/common/config/DefaultConfigManager.java

```diff
@@ -17,65 +17,116 @@
 package com.alipay.sofa.common.config;
 
 import com.alipay.sofa.common.utils.OrderComparator;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import com.google.common.util.concurrent.UncheckedExecutionException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.time.Duration;
 import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
 
 /**
  * @author zhaowang
  * @version : SofaCommonConfig.java, v 0.1 2020年10月20日 8:30 下午 zhaowang Exp $
  */
-public class DefaultConfigManger implements ConfigManager{
+public class DefaultConfigManager implements ConfigManager {
```

Comment:
Is it safe enough to change this public class name?

Reply:
DefaultConfigManager is used as a singleton in SofaConfigs, so it's ok to change the class name.